cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/Broker.java

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
This class should really be called `BrokerCapacity`. I know you have a class called that in the API module but we should call a spade a spade.